sport/app/football/implicits/Football.scala

```diff
@@ -44,8 +44,7 @@ trait Football {
       m.awayTeam.score,
     )
 
-    def isOn(date: LocalDate): Boolean =
-      m.date.toLocalDate.isAfter(date) && m.date.toLocalDate.isBefore(date.plusDays(1))
+    def isOn(date: LocalDate): Boolean = m.date.toLocalDate == date
```

Comment:
This looks equivalent to me, but just wondering if there's some reason it was written that way